tsup.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'tsup'
```

Comment:
Can we use `esbuild` instead?

Reply:
`tsup` is tailored towards Typescript and uses `esbuild` under the hood. Its the same thing without adding any more configurations.